modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -95,30 +115,36 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
             JOIN files f ON (f.SessionID=s.ID) 
             LEFT JOIN files_qcstatus fqc ON (fqc.FileID=f.FileID) 
             JOIN mri_acquisition_dates md ON (md.SessionID=s.ID)
-            LEFT JOIN (SELECT files.SessionID, MIN(QCLastChangeTime)
-            as QCLastChangeTime FROM files
-            LEFT JOIN files_qcstatus USING (FileID)
+            LEFT JOIN (SELECT files.SessionID, MIN(QCLastChangeTime) 
+            as QCLastChangeTime FROM files 
+            LEFT JOIN files_qcstatus USING (FileID) 
```

Comment:
GitHub is highlighting that there was a space added to the end of these lines; I don't think they're supposed to be there.